docs/understanding-tracking-design/versioning-your-data-structures/superseding-schema/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Superseding schemas"
+date: "2022-04-07"
+sidebar_position: 50
+---
+
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+```
+
+:::note
+
+You need >= Enrich 3.8.0 and >= Iglu Server 0.10.0 to be able to use this feature.
+
+No additional configuration change is required on Enrich or on Iglu Server to make superseding schema feature work.
+
+:::
+
+One of the most powerful features of Snowplow is schemas.
+
+Schemas define the structure of the data that you collect. Each schema defines what fields are recorded with each event that is captured, and provides validation criteria for each field. Schemas are also used to describe the structure of [entities that are attached to events](/docs/understanding-tracking-design/understanding-events-entities/index.md). You can get more information about schemas in [here](/docs/understanding-tracking-design/understanding-schemas-and-validation/index.md).
+
+We treat schemas as immutable objects because updating the schemas might make previously valid events invalid. However, there are some cases we want more flexibility. Let's go through an example. Let's say we have an mobile application. We are sending certain events from this application, and these events contain contexts with following schema:
+
+<details>
+  <summary>Geolocation 1-0-0</summary>
+  <CodeBlock>{`
```

Comment:
```suggestion
  <CodeBlock language="json">{`
```

And for all other relevant codeblocks. You could also swap out the tags with ```json but you'll need a blank line before and after otherwise it won't build right.